meshnets/main.py

```diff
@@ -54,12 +56,24 @@ def main(_):
         batch_size=3,
         shuffle=True,
     )
-    test_dataloader = DataLoader(test_dataset, batch_size=1, shuffle=True)
+    test_dataloader = DataLoader(test_dataset, batch_size=1, shuffle=False)
+
+    sample = dataset.get(0)
+    model = MeshGraphNet(node_features_size=sample.x.shape[1],
```

Comment:
Can you describe how these features appear?

I am having a hard time figuring out and need to go through all the files. It seems that sample is a graph of the dataset (one you process above), which already contains the features  we require. I think you can add briefly on top, what the model you are constructing here entails (features of nodes and edges).

Reply:
`sample` is indeed a sample of the dataset we constructed, where the dataset is composed of torch_geometric `Data` objects. What do you think I should add to clarify this?

Reply:
I was thinking of a description of the features and their shape. But maybe I am just being picky since this file can run for different data if we change the mesh to graphs. 

Keep it like this. But in the files loading the mesh see if more documentation is needed to understand this faster.